src/cephadm/cephadm

```diff
@@ -2291,6 +2295,24 @@ def command_adopt():
             call_throws(['umount', data_dir_src])
         os.rmdir(data_dir_src)
 
+        logger.info('Chowning content...')
+        call_throws(['chown', '-c', '-R', '%d.%d' % (uid, gid), data_dir_dst])
```

Comment:
This would work, but maybe a more pythonic approach using `os.walk` ?

https://kite.com/python/examples/4291/os-set-permissions-recursively-for-a-directory